### PR TITLE
Improved testing and time handling

### PR DESCRIPTION
**Pull request by mlange05**
This merge brings a few small bug fixes, improvements to the test base and automated handling of `datetime.timedelta` objects in the execution loop. In detail:
* Handle and interpret `datetime.timedelta` objects in the execution loop
* Use timedelta objects in example tests
* Add baseline tests for grid creation and field sampling functionalities
* Fix standalone `field` visualisation
* Allow setting `output_interval` without specifying an output file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,5 +35,5 @@ script:
     - flake8 tests
     - flake8 examples
     - python tests/test_peninsula.py --grid 20 10
-    - py.test -v -s tests/
+    - py.test tests/
```
Comment (erikvansebille) [inline .travis.yml]: Why would we want less verbosivity here? I liked being able to see which tests worked and which didn't. Is there a reason for this?

File: parcels/particle.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from collections import OrderedDict, Iterable
 import matplotlib.pyplot as plt
 import math
-import datetime
+from datetime import timedelta
```
Comment (erikvansebille) [inline parcels/particle.py]: Since we import `timedelta` in the tests as `delta`, should we do the same here, for consistency?

**Comment by erikvansebille** [comment]
Apart from two minor issues (level of verbosivity in test.py and whether to import `timedelta` as `delta` or not), I think this is a very nice housekeeping PR! Cleans up some of the code!

File: .travis.yml (hunk 1 above)
Comment (mlange05) [inline .travis.yml]: Hehe, I was getting annoyed with reading so many compilation messages, but sure we can drop this change until our test base has grown a bit further.

**Comment by mlange05** [comment]
Cool, addressed comments, rebased and green travis. This should be good to go.

**Pull request merged**